src/deploy/functions/cache/applyHash.ts

```diff
@@ -28,13 +36,18 @@ export function applyBackendHashToBackends(
 function applyBackendHashToEndpoints(
   wantBackend: Backend,
   envHash: string,
+  codebaseFilters: EndpointFilter[],
```

Comment:
nit* I wonder if more appropriate var name for this is "deploy filter" or something - not sure what you mean by codebaseFilters here.

Reply:
Fair point. I wanted to convey that it was filtered by codebase already... but thats an implementation detail that the function doesn't need to call out. 👍 Renamed back to `filters`.